cypress/integration/lab.test.js

```diff
@@ -0,0 +1,45 @@
+const deckId = "my-deck-id";
+
+const clickForDeck = (fixture, count = 5) => {
+  cy.intercept(`https://deckofcardsapi.com/api/deck/${deckId}/draw/?count=${count}`, { fixture });
+  cy.get("button").click();
+};
+
+const visitWithFirstDeck = () => {
+  cy.intercept("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1", {
+    deck_id: deckId,
+  }).as("newDeck");
```

Comment:
How is the alias used here? Not blocking, just a question for my own education.

Reply:
Ah, unused code, I'd forgotten to remove it.